README.md

```diff
@@ -65,7 +65,7 @@
 
 ### Key Features <a name="key-features"></a>
 
```

Comment:
Good job on the improvements! The transaction button is now in the correct place. However, as mentioned by the previous reviewer, it's important to ensure that each transaction card matches the design requirements. 

- [ ] Even though there may not be an icon for the transactions, you can consider using a placeholder icon or displaying the cards without an icon while making sure that the title, date, and amount follow the specified style. Additionally, make sure that the overall card style aligns with the design requirements.
With icon:

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Samuelwanza/financier/assets/55840999/72afd80a-4580-4f95-b4e0-e11f8ec5ca6b)| ![image](https://github.com/Samuelwanza/financier/assets/55840999/8d07b4c8-8419-431e-8c25-949241ab11ab) |

Without icon:

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Samuelwanza/financier/assets/55840999/72afd80a-4580-4f95-b4e0-e11f8ec5ca6b)|![image](https://github.com/Samuelwanza/financier/assets/55840999/6fdb247a-75c0-4eed-a62d-775c8d3fb90e)|



By maintaining consistency in the styling of transaction cards, you'll achieve a more polished and cohesive look for your application. Keep up the good work! 🚀🎨